app/selenium_ui/conftest.py

```diff
@@ -28,19 +28,19 @@
 
 def __get_current_results_dir():
     if 'TAURUS_ARTIFACTS_DIR' in os.environ:
-        return os.environ.get('TAURUS_ARTIFACTS_DIR')
+        return Path(os.environ.get('TAURUS_ARTIFACTS_DIR'))
     else:
         # TODO we have error here if 'results' dir does not exist
         results_dir_name = datetime.datetime.now().strftime("%Y-%m-%d_%H-%M-%S")
-        pytest_run_results = f'results/{results_dir_name}_local'
-        os.mkdir(pytest_run_results)
+        pytest_run_results = Path(f'results/{results_dir_name}_local')
+        os.makedirs(pytest_run_results, exist_ok=True)
```

Comment:
what do you think about `Path("/my/directory").mkdir(parents=True, exist_ok=True)`?